allo/harness/vivado/run.tcl

```diff
@@ -21,7 +21,7 @@ add_files -tb host.cpp -cflags "-std=gnu++0x"
 
 open_solution "solution1"
 # Use Zynq device
-set_part {xc7z020clg484-1}
+set_part {xczu3eg-sbva484-1-i}
```

Comment:
We will be providing an interface for passing in the target FPGA board name, so please keep this tcl file unchanged in this PR.